explainer-client/src/main/scala/CAPIConfig.scala

```diff
@@ -0,0 +1,8 @@
+
+case class TagObject(id: String, webTitle: String)

```

Comment:
I'd put this file under src/main/models/Models.scala, and with my suggested changes TagObject is the only thing you'll need in it.